model/cloud.go

```diff
@@ -42,8 +42,50 @@ const (
 	SubscriptionFamilyOnPrem = SubscriptionFamily("on-prem")
 )
 
-const defaultCloudNotifyAdminCoolOffDays = 30
-const CloudNotifyAdminInfo = "cloud_notify_admin_info"
+type MattermostPaidFeature string
+
+const (
+	PaidFeatureGuestAccounts           = MattermostPaidFeature("mattermost.feature.guest_accounts")
+	PaidFeatureCustomUsergroups        = MattermostPaidFeature("mattermost.feature.custom_user_groups")
+	PaidFeatureCreateMultipleTeams     = MattermostPaidFeature("mattermost.feature.create_multiple_teams")
+	PaidFeatureStartcall               = MattermostPaidFeature("mattermost.feature.start_call")
+	PaidFeaturePlaybooksRetrospective  = MattermostPaidFeature("mattermost.feature.playbooks_retro")
+	PaidFeatureUnlimitedMessages       = MattermostPaidFeature("mattermost.feature.unlimited_messages")
+	PaidFeatureUnlimitedFileStorage    = MattermostPaidFeature("mattermost.feature.unlimited_file_storage")
+	PaidFeatureUnlimitedIntegrations   = MattermostPaidFeature("mattermost.feature.unlimited_integrations")
+	PaidFeatureUnlimitedBoardcards     = MattermostPaidFeature("mattermost.feature.unlimited_board_cards")
+	PaidFeatureAllProfessionalfeatures = MattermostPaidFeature("mattermost.feature.all_professional")
+	PaidFeatureAllEnterprisefeatures   = MattermostPaidFeature("mattermost.feature.all_enterprise")
+)
+
+type CloudSKU string
+
+const (
+	CloudSKUStarter      = CloudSKU("cloud-starter")
+	CloudSKUProfessional = CloudSKU("cloud-professional")
+	CloudSKUEnterprise   = CloudSKU("cloud-enterprise")
+)
```

Comment:
We could use this license skus to be generic that are the same for cloud/self-managed under `shortNameSku` https://github.com/mattermost/mattermost-server/blob/7deb408898aaf88aa4644652d7f250a26fb5bdbc/model/license.go#L24-L25